repl/scala-2.11/src/main/scala/org/apache/spark/repl/Main.scala

```diff
@@ -44,7 +44,9 @@ object Main extends Logging {
 
   private def scalaOptionError(msg: String): Unit = {
     hasErrors = true
+    //scalastyle:off
```

Comment:
Is this needed / related? I thought we maybe didn't check style on this file because it's copied from scala